Example/URBNConvenience/SwiftVC.swift

```diff
@@ -41,6 +42,11 @@ class SwiftVC: UIViewController {
             "H:|-[redView]-|",
             views: views
         )
+        
+        imageView.image = qrImage("core image supports qr codes", foregroundColor: .purpleColor(), backgroundColor: .greenColor(), size: CGSizeMake(100.0, 100.0))
```

Comment:
I'm wondering because of the complexity with this method should the first param be a named one?